genie-web/src/test/java/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.configs;
+
+import com.netflix.genie.core.properties.DataServiceRetryProperties;
+import com.netflix.genie.core.properties.HealthProperties;
+import com.netflix.genie.core.properties.JobsProperties;
+import com.netflix.genie.test.categories.IntegrationTest;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * Integration test for PropertiesConfig values binding.
+ *
+ * @author mprimi
+ * @since 3.1.0
+ */
+@Category(IntegrationTest.class)
+@RunWith(SpringRunner.class)
+@TestPropertySource(locations = "classpath:/com/netflix/genie/web/configs/PropertiesConfigIntegrationTest.properties")
```

Comment:
You could likely also accomplish this (with a yml file as well) by just putting a properties file in source/test/resources and calling it like `application-propstest.yml` and then activating the `@Profile("propstest")` for this test. That should load that properties source. Also I wouldn't put this properties file explicitely in this location as it inconsistent with the rest of our locations.